framework/db/sqlite/conditions/InConditionBuilder.php

```diff
@@ -43,6 +43,9 @@ protected function buildCompositeInCondition($operator, $columns, $values, &$par
         foreach ($values as $value) {
             $vs = [];
             foreach ($columns as $i => $column) {
+                if (strpos($column, '.') !== false && !isset($value[$column])) { // if column have alias and is not indexed with alias, remove it
+                    $column = substr($column, strpos($column, '.') + 1); 
```

Comment:
or use `list($column) =explode(',', $column, 2);`

Reply:
changed to list(, $column) = explode('.', $column, 2);

Reply:
@ntesic not complete implementation rewrite to 
```php
if (!isset($value[$column])) {
   list(, $column) = explode('.', $column, 2) + [1 => $column];
}
```

Reply:
Ok, didn't understand that want to totally omit usage of strpos and substr. Ok, I will rewrite it

Reply:
@ntesic  I'm didn't
> **or** use ...